src/Icons/doc/index.rst

```diff
@@ -89,40 +88,25 @@ Icon Sets
 
 There are many icon sets available, each with their own unique style and set of
 icons, providing a wide range of icons for different purposes, while maintaining
-a consistent look and feel across your application.
-
-With more than 200,000 icons available, you can find the perfect icon for your
-project. Here are some of the most popular icon sets available:
-
-========================= ======= ========== =========== =================
-Icon Set                    Icons License    Prefix      Example
-========================= ======= ========== =========== =================
-`Bootstrap Icons`_           2000 MIT        bi          bi:check
-`Boxicons`_                  1700 MIT        bx          bx:check
-`Flowbite`_                  1000 MIT        flowbite    flowbite:check
-`FontAwesome Icons`_         7000 CC BY 4.0  fa6-regular fa6-regular:check
-`Heroicons`_                  300 MIT        heroicons   heroicons:check
-`Iconoir`_                   1500 MIT        iconoir     iconoir:check
-`Ionicons`_                  1300 MIT        ion         ion:check
-`Lucide Icons`_              1500 MIT        lucide      lucide:check
-`Material Design Icons`_     5000 Apache 2   mdi         mdi:check
-`Octicons`_                   600 MIT        octicon     octicon:check
-`Phosphor Icons`_            7000 MIT        ph          ph:check
-`Tabler Icons`_              5200 MIT        tabler      tabler:check
-========================= ======= ========== =========== =================
-
-- `Bootstrap Icons <https://icons.getbootstrap.com/>`_
```

Comment:
These links still need to exist for the table below, no?

Reply:
You are right ... the list of links should be fixed a bit, not removed. I re-added it at the bottom of the article. Thanks!